api/tests/unit/organisations/test_unit_organisations_tasks.py

```diff
@@ -247,6 +247,8 @@ def test_handle_api_usage_notifications_below_100(
         "organisations.tasks.get_current_api_usage",
     )
     mock_api_usage.return_value = 91
+    mocker.patch("organisations.tasks.get_client")
```

Comment:
Should we test what happens when the flag is enabled / disabled?

Reply:
Sure I've added a new test for when the flag is disabled and reworked some of the mock logic to return the actual expected feature flag value.